### PR TITLE
Pick-and-Learn

### PR DESCRIPTION
**Pull request by nathanpainchaud**
A bit of a puzzling paper seen at MICCAI 2019. Some of the implementation details are unclear, but the results are promising.

### CODE REVIEW

File: article/_posts/2019-10-16-PickAndLearn.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+The resulting loss used by the segmentation model then becomes:
+
+![](/article/images/PickAndLearn/equation1.png)
+
+The authors mention that they apply a softmax (which does not seem to be pictured in the equation) to rescale all the
+weights between 0 and 1.
+
+
+## Overfitting Control Module (OCM)
+The motivation behind the ÒCM is that overfitting problems can arise from the use of the QAM, if the QAM has a larger
```
Comment (AntoineTheb) [inline article/_posts/2019-10-16-PickAndLearn.md]: ÒCM -> OCM

File: article/_posts/2019-10-16-PickAndLearn.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+loss and U-Net as their segmentation network.
+
+![](/article/images/PickAndLearn/figure2.png)
+
+
+## Quality Awareness Module (QAM)
+The QAM runs parallel with the segmentation module to calculate weight values to apply to the loss from each sample.
+During training, the re-weighted loss is used for both the segmentation module and the QAM.
+
+The network used for the QAM is a VGG network, where the number of input channel is change to $$n + 1$$, where $$n$$ is
```
Comment (jhlegarreta) [inline article/_posts/2019-10-16-PickAndLearn.md]: the number of input channel : the number of input channels

File: article/_posts/2019-10-16-PickAndLearn.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+
+## Overfitting Control Module (OCM)
+The motivation behind the ÒCM is that overfitting problems can arise from the use of the QAM, if the QAM has a larger
+weight on a small subset with clear-labeled samples. The authors also state that if the QAM makes mistakes during the
+training process, the relative weight of this sample can be too small or too big for the network to correct.
+
+To counter this, the authors add an extra restriction between the quality awareness module and the softmax layer. They
+simply use a weighted hyperbolic tangent (as shown in the equation below), to limit the relative ratio of the output
+quality score from (−∞,∞) to (−λ,λ). The authors state they empirically set λ to be 2.
```
Comment (jhlegarreta) [inline article/_posts/2019-10-16-PickAndLearn.md]: Can math symbols be written using LaTeX syntax?

File: article/_posts/2019-10-16-PickAndLearn.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+![](/article/images/PickAndLearn/equation2.png)
+
+
+## Data
+The authors tested their method on the public JSRT dataset, which contains 247 X-rays images of the chest. The labeled
+structures are the heart, clavicles, and lungs.
+
+To generate noisy-labeled images, the authors used different types and levels of noises. They randomly selected
+0%, 25%, 50% and 75% samples from the training set and eroded or dilated them with 1 ≤ n i ≤ 8 and 5 ≤ n i ≤ 13 pixels.
```
Comment (jhlegarreta) [inline article/_posts/2019-10-16-PickAndLearn.md]: Can math symbols be written using LaTeX syntax?

File: article/_posts/2019-10-16-PickAndLearn.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+![](/article/images/PickAndLearn/figure3.png)
+
+Figure 4 gives some insight into how the QAM affects the final loss during the training.
+
+![](/article/images/PickAndLearn/figure4.png)
+
+
+# Remarks
+- Approximative English and figure make it unclear whether the QAM uses the original segmentation loss or the
```
Comment (jhlegarreta) [inline article/_posts/2019-10-16-PickAndLearn.md]: figure : figures

 Approximative English : Unclear wording at some points (?)

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 1 above)
Comment (nathanpainchaud) [reply article/_posts/2019-10-16-PickAndLearn.md]: Typo corrected!

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 2 above)
Comment (nathanpainchaud) [reply article/_posts/2019-10-16-PickAndLearn.md]: Typo corrected!

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 3 above)
Comment (nathanpainchaud) [reply article/_posts/2019-10-16-PickAndLearn.md]: I wrote this review while I had no internet connection while at airports in China, so I couldn't look up the names of the symbols, and forgot to edit my review before uploading it. Thanks for the catch, and I'll make sure not to forget it next time!

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 4 above)
Comment (nathanpainchaud) [reply article/_posts/2019-10-16-PickAndLearn.md]: Symbols changed to LaTeX. Also quick question, but would percentage (%) be considered a math symbol? I kept it as is for now, but I could change it if required.

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 5 above)
Comment (nathanpainchaud) [reply article/_posts/2019-10-16-PickAndLearn.md]: I reworked my remark to use your suggestion and to specify which single figure in particular I was referring to.

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-10-16-PickAndLearn.md]: Done in 6b69fdd.

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 4 above)
Comment (jhlegarreta) [reply article/_posts/2019-10-16-PickAndLearn.md]: Done in 6b69fdd.

As for the percentage symbol questions, I think it is a matter of preferences. It definitely is a maths symbol, but I think it is fine to write them as plain text.

File: article/_posts/2019-10-16-PickAndLearn.md (hunk 5 above)
Comment (jhlegarreta) [reply article/_posts/2019-10-16-PickAndLearn.md]: Done in 6b69fdd. It's very clear now. Thanks @nathanpainchaud.

**Pull request merged**